### PR TITLE
chore(deps): Updated @newrelic/security-agent to v1.2.0

### PR DESCRIPTION
**Pull request by sumitsuthar**
#### Features
* Added instrumentation for express framework's res.download() and res.sendFile()
#### Bug fixes
* Handling to decrypt fuzz header data for IAST scanning 
* Logging and snapshot file fixes 
#### Miscellaneous chores
* Prepend vulnerability case type with apiId 
* Updated jsonVersion to v1.2.0 
* Bumped undici from 5.28.3 to 5.28.4

## How to Test

`npm run versioned:security`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -196,7 +196,7 @@
     "@newrelic/aws-sdk": "^7.3.0",
     "@newrelic/koa": "^9.1.0",
     "@newrelic/ritm": "^7.2.0",
-    "@newrelic/security-agent": "^1.1.1",
+    "@newrelic/security-agent": "^1.2.0",
```
Comment (jsumners-nr) [inline package.json:199]:
```suggestion
    "@newrelic/security-agent": "1.2.0",
```

We'd like to pin this dependency going forward. Please remove the `^` qualifier.

**Review by jsumners-nr — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (bizob2828) [reply package.json:199]: this is ok but @sumitsuthar we don't need PRs when you release unless it's a semver major.  The `^1.x.x` will include all releases in 1.x

**Review by bizob2828 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (sumitsuthar) [reply package.json:199]: Got it.

**Comment by sumitsuthar** [comment]
As we don't need PRs unless it's a semver major. The ^1.x.x will include all releases in 1.x. Hence closing the PR.

**Pull request closed without merging**